auth/api/logging.go

```diff
@@ -28,314 +27,533 @@ func LoggingMiddleware(svc auth.Service, logger *slog.Logger) auth.Service {
 
 func (lm *loggingMiddleware) ListObjects(ctx context.Context, pr auth.PolicyReq, nextPageToken string, limit int32) (p auth.PolicyPage, err error) {
 	defer func(begin time.Time) {
-		message := fmt.Sprintf("Method list_objects took %s to complete", time.Since(begin))
+		args := []interface{}{
+			slog.String("duration", time.Since(begin).String()),
+			slog.Int64("limit", int64(limit)),
+		}
 		if err != nil {
-			lm.logger.Warn(fmt.Sprintf("%s with error: %s.", message, err))
+			args = append(args, slog.Any("error", err))
+			lm.logger.Warn("List objects failed to complete successfully", args...)
 			return
 		}
-		lm.logger.Info(fmt.Sprintf("%s without errors.", message))
+		lm.logger.Info("List objects completed successfully", args...)
 	}(time.Now())
 
 	return lm.svc.ListObjects(ctx, pr, nextPageToken, limit)
 }
 
 func (lm *loggingMiddleware) ListAllObjects(ctx context.Context, pr auth.PolicyReq) (p auth.PolicyPage, err error) {
 	defer func(begin time.Time) {
-		message := fmt.Sprintf("Method list_all_objects took %s to complete", time.Since(begin))
+		args := []interface{}{
+			slog.String("duration", time.Since(begin).String()),
+			slog.Group(
+				"object",
+				slog.String("type", pr.ObjectType),
+			),
+			slog.Group(
+				"subject",
+				slog.String("id", pr.Subject),
+				slog.String("type", pr.SubjectType),
```

Comment:
Let's flatten Policy by using a single Group here, like:
```suggestion
			slog.Group(
				"policy_request",
				slog.String("object_type", pr.ObjectType),
				slog.String("subject_id", pr.Subject),
				slog.String("subject_type", pr.SubjectType))
```